includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -432,6 +432,18 @@ protected function get_posted_fields() {
 				$field_type = str_replace( '-', '_', $field['type'] );
 				$handler    = apply_filters( "job_manager_get_posted_{$field_type}_field", false );
 
+				$this->fields[ $group_key ][ $key ]['empty'] = false;
+
+				if ( ! isset( $field['before_sanitize'] ) ) {
+					$field['before_sanitize'] = function ( $value ) use ( $group_key, $key ) {
+						if ( is_string( $value ) ) {
+							$value = trim( $value );
+						}
+
+						$this->fields[ $group_key ][ $key ]['empty'] = empty( $value );
```

Comment:
Won't this report more values that it should as empty? We were checking for null only previously.

Reply:
This is essentially just an earlier check from what happens in `\WP_Job_Manager_Form_Submit_Job::validate_fields` for the `x is a required field`, which uses and has always used just `empty()`.

Reply:
Cool! Missed that!